cpu/stm32/periph/spi.c

```diff
@@ -134,12 +134,20 @@ void spi_init_pins(spi_t bus)
     gpio_init_af(spi_config[bus].mosi_pin, GPIO_AF_OUT_PP);
     gpio_init(spi_config[bus].miso_pin, GPIO_IN);
 #else
```

Comment:
you should also update the `CPU_FAM_STM32F1` case

Reply:
Done